samples/net/openthread/ot_cli/prj.conf

```diff
@@ -0,0 +1,59 @@
+# Generic networking options
+CONFIG_NETWORKING=y
+CONFIG_NET_UDP=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_SOCKETS=y
```

Comment:
Do we really need sockets here? They're not used by the sample (there might be some hidden dependencies I'm not aware of though).